packages/dev-utils/src/rpc.ts

```diff
@@ -41,7 +41,7 @@ export class RPC {
             method,
             params,
         });
-        this._url += 1;
+        this._id += 1;
```

Comment:
@BMillman19 this happened during the big `underscore for private methods` refactor you undertook. Any ideas how this could have happened?